common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -1949,7 +1961,27 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "the HTTP client."),
     HIVE_DRUID_HTTP_READ_TIMEOUT("hive.druid.http.read.timeout", "PT1M", "Read timeout period for the HTTP\n" +
         "client in ISO8601 format (for example P2W, P3M, PT1H30M, PT0.750S), default is period of 1 minute."),
-
+    HIVE_DRUID_BASE_PERSIST_DIRECTORY("hive.druid.basePersisitDirectory", "/tmp",
+            "local base persist directory used while indexing  data"
```

Comment:
@b-slim , can you use natural language + punctuation in all property descriptions? Just trying to anticipate Lefty's comments ;)